src/reference/asciidoc/overview.adoc

```diff
@@ -216,7 +216,7 @@ IMPORTANT: Starting with _version 4.0_, the `@EnableIntegration` annotation has
 registration of Spring Integration infrastructure beans (see
 http://docs.spring.io/spring-integration/docs/latest-ga/api/org/springframework/integration/config/EnableIntegration.html[JavaDocs]).
 This annotation is required when only Java & Annotation configuration is used, e.g. with Spring Boot and/or
-Spring Integration Messaging Annotation support and Spring Integration Java DSL with no XML integration configuration.
+Spring Integration Messaging Annotations support and Spring Integration Java DSL with no XML integration configuration.
```

Comment:
You undid my polish :smile: and made it plural again.

In this context, it should be singular - don't ask to me to explain the quirks of English.